client/extensions/woocommerce/state/action-types.js

```diff
@@ -1,3 +1,7 @@
+export const WOOCOMMERCE_API_SET_ERROR = 'WOOCOMMERCE_API_SET_ERROR';
+export const WOOCOMMERCE_API_CLEAR_ERROR = 'WOOCOMMERCE_API_CLEAR_ERROR';
+export const WOOCOMMERCE_API_CREATE_PRODUCT = 'WOOCOMMERCE_API_CREATE_PRODUCT';
```

Comment:
Hmm, that might be a good thing to do. But if we decide to do that, it should probably be a separate PR just to rename actions.